posts/affine-group-scheme-hopf-algebra.md

```diff
@@ -0,0 +1,266 @@
+
+A special class of varieties, namely *affine varieties*,
+can be constructed more readily using the *Spec functor*,
+taking a commutative ring to a variety its *prime spectrum*.
+The previous sentence is a bit scary,
+but all we need to know is that if $R$ and $S$ are commutative rings,
+then morphisms $\mathrm{Spec}\ R \to \mathrm{Spec}\ S$
+exactly correspond to ring homomorphisms $S \to R$.
+
+We can reinterpret our variety $\mathbb C^n$ as $\mathrm{Spec}\ \mathbb C[X_1, \dots, X_n]$,
+the spectrum of the polynomial ring in $n$ variables.
+Similarly, $(\mathbb C^\times)^n$ is the same variety as $\mathrm{Spec}\ \mathbb C[X_1^\pm, \dots, X_n^\pm]$,
+the spectrum of multivariate Laurent series in $n$ variables.
+The natural inclusion $\mathbb C[X_1, \dots, X_n] \hookrightarrow \mathbb C[X_1^\pm, \dots, X_n^\pm]$
+corresponds to the natural inclusion $(\mathbb C^\times)^n \hookrightarrow \mathbb C^n$.
+
+# Algebraic groups
+
+Recall the definition of a group. A *group structure* on a set $G$ consists of:
+1. An **identity** $e \in G$.
+2. A **multiplication** $(\cdot * \cdot) : G \times G \to G$.
+3. An **inverse** $(\cdot^{-1}) : G \to G$.
+
+respecting the following *group axioms*:
+1. For all $g \in G$, $e * g = g$ (and $g * e = g$).
+2. For all $g, h, k \in G$, $(g * h) * k = g * (h * k)$.
+3. For all $g \in G$, $g^{-1} * g = e$ (and $g * g^{-1} = e$).
+
+A group structure can be described purely in terms of maps between
+$G, G \times G$ and the one element set $\{*\}$,
+meaning that we can reinterpret groups in any category $\mathsf{C}$
+where $(\cdot \times \cdot)$ and $\{*\}$ make sense,
+i.e. in any category with a binary product $(\cdot \otimes \cdot)$ and terminal object $\mathbf{1}_C$
+A **group object** $G \in \mathsf{C}$ then consists of:
+1. A **unit morphism** $\eta : \mathbf{1}_C \to G$.
+2. A **multiplication morphism** $\mu : G \otimes G \to G$.
+3. An **inverse morphism** $\iota : G \to G$.
+
+making the following diagrams commute:
+<span style="color:red">**TODO: Add diagrams**</span>
+
+If the following further diagram commutes, then we have a **commutative group object**:
+<span style="color:red">**TODO: Add diagram**</span>
+
+Analogously to group homomorphisms, if $G$ and $H$ are group objects,
+then a morphism $f : G \to H$ is a **group morphism** if the following two diagrams commute:
+<span style="color:red">**TODO: Add diagrams**</span>
+
+A group object in the category of varieties is called an *algebraic group*.
+In other categories of interest, such as the categories of sets/topological spaces/smooth manifolds,
+group objects recover the notions of a group/topological group/Lie group.
+
+Beware that those examples are misleading in that topological groups and Lie groups are
+honest-to-god groups with extra structure.
+This elementwise interpretation of a group object completely breaks down in categories
+such as the category of schemes where there is no notion of "elements".
```

Comment:
I suggest to ditch this portion. This is still wrong. There is still a valid notion of elements (or points) on schemes, and group schemes are exactly groups (on the points) satisfying some property (e.g. multiplication being polynomials, and some other base-change compatible conditions) just like Lie groups are groups (on the points of the manifold) satisfying some property (e.g. multiplication being smooth).